cluster/charts/crossplane-types/crds/workload.crossplane.io_kubernetestargets.yaml

```diff
@@ -0,0 +1,145 @@
+
```

Comment:
why is this one showing up as an add but the other ones are showing up as renames?

the other "renamed" CRDs will be committed in both places instead of being renamed after https://github.com/crossplaneio/crossplane/pull/1139/files#r371825341 is resolved, right?

Reply:
This file was not available when I started this PR and introduced recently. I believe it is related to this as well as we were gitignoring main CRD dir with this PR (which will be changed with  https://github.com/crossplaneio/crossplane/pull/1139/files#r371825341).

> the other "renamed" CRDs will be committed in both places instead of being renamed after https://github.com/crossplaneio/crossplane/pull/1139/files#r371825341 is resolved, right?

Yes